index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h3>Hello World, we are software developers!</h3>
+
+      <div id="main">
+        <h3 id="title">Dr.Norman Borlug</h3>
+          <div id="img-div">
+            <!---img here-->
+            <img src="https://img.freepik.com/free-photo/pleased-young-female-doctor-wearing-medical-robe-stethoscope-around-neck-standing-with-closed-posture_409827-254.jpg?w=2000" alt="doctor" id="image">
+            <div id="image-caption">Caption Here</div>
+          </div>
+          <div id="tribute-info">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Distinctio, perferendis excepturi ratione facilis a temporibus dolores soluta repellendus aspernatur cum impedit necessitatibus beatae aperiam aliquid vel omnis! Laboriosam, repudiandae libero?</div>
+          <a id="tribute-link" href="www.google.com" target="blank">tribute link</a>
+      </div>
+
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
I suggest removing the extra comments in the code. This is so as to make the code clean.